### DISCUSSION TITLE
How to translate to BCC?

### DISCUSSION THREAD

**Discussion by akshatagarwl** [bpftrace/bpftrace#2650] [Development]
Is there a way I can get the underlying generated BCC program that actually runs when I run the bpftrace programs.
If there is no way to get this is there a general rule that applies in order to infer what kind of programs use what kind of maps/helpers. Where should I look in the code for this?

**Comment by fbs** [comment]
what is a 'bcc program'? We compile to llvm ir which compiles down to ebpf bytecode. You can use `--emit-elf` or `bpftool prog dump` to see the generated program
